fiftyone/core/dataset.py

```diff
@@ -4525,6 +4525,10 @@ def save_view(
         self._doc.last_modified_at = now
         self._doc.save(virtual=True)
 
+        view._set_name(name)
+        if view._is_generated:
+            view._dataset.persistent = True
```

Comment:
this is going to mean some api permissions work is necessary for teams

Reply:
Yep I have a plan for that. Can chat offline